batch-setup/make_meta_tiles.py

```diff
@@ -281,33 +357,113 @@ def _big_jobs(rawr_bucket, prefix, key_format_type, rawr_zoom, group_zoom,
         for y in range(num_coords):
             coord = Coordinate(zoom=group_zoom, column=x, row=y)
             tasks.append(p.apply_async(job_sizer, (coord,)))
+            grouping_queue.put_nowait(coord)  # queue for future size counting
 
         # collect tasks and put them into the big jobs list.
         for task in tasks:
-            coord, size = task.get()
-            if size >= size_threshold:
-                big_jobs[coord] = True
+            coord, sizes = task.get()
+            all_sizes.update(sizes)
+
+    # now use all_sizes plus the size_threshold to find the lowest zoom we can group each coordinate into
```

Comment:
Can you add another line or two of comment here? My intuition is that we're trying to group tiles by estimated memory usage so that we get to but don't go beyond a target usage. What you have here is good, but maybe needs the extra bit of info about why grouping is happening.